resolver/map.jinja

```diff
@@ -8,6 +8,9 @@
         'Debian': {
             'conf_path': '/etc/resolvconf/run/resolv.conf',
         },
+        'SUSE': {
```

Comment:
```suggestion
        'Suse': {
```

In order to maintain consistency with other formulas, please consider the suggestion above.  Refer back to the following search for more details:

* https://github.com/search?l=HTML%2BDjango&q=org%3Asaltstack-formulas+SUSE&type=Code

Reply:
@cschneemann Of course, please confirm that works at your end first.